tools/CoqMakefile.in

```diff
@@ -719,6 +726,10 @@ endif
 $(VOFILES): %.vo: %.v
 	$(SHOW)COQC $<
 	$(HIDE)$(TIMER) $(COQC) $(COQDEBUG) $(TIMING_ARG) $(COQFLAGS) $(COQLIBS) $< $(TIMING_EXTRA)
+ifeq '$(COQMF_COQ_NATIVE_COMPILER_DEFAULT)' "yes"
+	$(SHOW)COQNATIVE $*.vo
+	$(HIDE)$(COQNATIVE) $(COQLIBS) $*.vo
```

Comment:
I think it would be nice to allow timing for these files.  Maybe add something like
```make
TIMECMD_NATIVE?=$(TIMECMD)
# Use command time on linux, gtime on Mac OS
TIMEFMT_NATIVE?="$*.cmxs (real: %e, user: %U, sys: %S, mem: %M ko)"
ifneq (,$(TIMED))
ifeq (0,$(shell command time -f "" true >/dev/null 2>/dev/null; echo $$?))
STDTIME_NATIVE?=command time -f $(TIMEFMT_NATIVE)
else
ifeq (0,$(shell gtime -f "" true >/dev/null 2>/dev/null; echo $$?))
STDTIME_NATIVE?=gtime -f $(TIMEFMT_NATIVE)
else
STDTIME_NATIVE?=command time
endif
endif
else
STDTIME_NATIVE?=command time -f $(TIMEFMT_NATIVE)
endif
```
and then add
```make
TIMER_NATIVE=$(if $(TIMED), $(STDTIME_NATIVE), $(TIMECMD_NATIVE))
```
and then use
```suggestion
	$(HIDE)$(TIMER_NATIVE) $(COQNATIVE) $(COQLIBS) $*.vo
```